dataclients/kubernetes/update_test.go

```diff
@@ -25,9 +27,7 @@ func TestUpdateOnlyChangedRoutes(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if len(r) != 1 || r[0].Id != healthcheckRouteID {
-		t.Fatal("no healthcheck route received")
-	}
+	assert.EqualValues(t, healthcheckRoutes(false), r, "no healthcheck routes received")
```

Comment:
For one check add a dependency?
I would rather like to remove all the testify dependencies.

Reply:
fine for me if it prints the nested things as diff well.
I just don't like the helper library, because you can write a little more code that is understandable without it.